src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -588,26 +588,19 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environmental and recording system's
-noise.
-In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
-each experimental session.
-It is RECOMMENDED to store the empty-room recording inside a subject directory
-named `sub-emptyroom`.
-The label for the [`task-<label>`](../99-appendices/09-entities.md#task) entity in the empty-room recording SHOULD be
-set to `noise`.
-If a [`session-<label>`](../99-appendices/09-entities.md#ses) entity is present, its label SHOULD be the date of the
-empty-room recording in the format `YYYYMMDD`, that is `ses-YYYYMMDD`.
-The `scans.tsv` file containing the date and time of the acquisition SHOULD
-also be included.
-The rationale is that this naming scheme will allow users to easily retrieve the
-empty-room recording that best matches a particular experimental session, based
-on date and time of the recording.
-It should be possible to query empty-room recordings just like usual subject
-recordings, hence all metadata sidecar files (such as the `channels.tsv`) file
-SHOULD be present as well.
+Empty-room MEG recordings capture the environmental and recording system's noise.
+In the context of BIDS it is RECOMMENDED to perform an empty-room recording for each experimental session.
 
-Example:
+If empty-room recordings are not collected for each individual experimental session, one empty-room recording may be used with multiple subjects and/or sessions.
+For example, they may be collected once a day.
+In that case it is RECOMMENDED to store the empty-room recording inside a subject directory named `sub-emptyroom`.
```

Comment:
"In that case": does that imply that in the former case, it is RECOMMENDED to store it in the subject folder together with the other tasks? If this is the case, maybe it should be made more explicit?

Would it make sense to separate the two typical cases even more clearly, with headers: 
 - case 1: one noise recordings per session folder
 - example 1
 - case 2: one noise recordings per day
 - example 2
?

Reply:
> Would it make sense to separate the two typical cases even more clearly, with headers?

I like this idea.

Reply:
An example 3 could illustrate the case when there is a conflict between a local noise recording in the session, and a sub-emptyroom. Automatically assigning the correct noise recording to a given task recording needs arbitration.

A policy like "if there is no noise recording in a session, and several noise recordings are present in sub-emptyroom for a given day, then the noise recording closest in time should be assigned to the session's recordings".

Reply:
BIDS uses the 80/20 Pareto Principle: we don't have to deal with all possible situations that might arise (as then we'll continue discussing indefinitely), but only with those 20% of the cases that result in 80% of the value. I think your case 3 falls outside the 80% of the value.

Reply:
thanks for the suggestion for splitting up the examples @dnacombo. With commit 0aa328e85799aa5f0a951545c8838fddc7ee1eba I have made the two cases more explicit in the text as well.